.github/workflows/ci.yml

```diff
@@ -104,6 +104,7 @@ jobs:
         -DF3D_MODULE_ASSIMP=ON
         -DF3D_MODULE_ALEMBIC=ON
         -DF3D_PYTHON_BINDINGS=ON
+        -DF3D_INSTALL_SDK=ON
```

Comment:
not needed in android ?

Reply:
No. The Android workflow generate a `f3d-java` library for the 4 architectures and it is used as a dynamic library through JNI.